src/main/java/Group15/View/EditWorkoutView.java

```diff
@@ -46,7 +46,7 @@ public static Scene createScene(Workout workout)
     private static Pane createTitlePane()
     {
         HBox titlePane = new HBox();
-        titlePane.setAlignment(Pos.TOP_CENTER);
+        titlePane.setAlignment(Pos.TOP_LEFT);
```

Comment:
Do you not test your code at all? Please run the code and take a look at whats wrong with the alignment